### PR TITLE
add gpu metric display in top job

### PR DESCRIPTION
**Pull request by xiaozhouX**
Add GPU metric from prometheus display in arena top job.

```
# arena top job
NAME                           STATUS   TRAINER  AGE  NODE           GPU(Requests)  GPU(Allocated)  GPU(Duty Cycle)  GPU(Memory Usage)
tf-dist-data-auto-scaler-test  RUNNING  TFJOB    1h   192.168.0.175  1              1               0.000000         0.000082
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/arena/74)

### CODE REVIEW

**Comment by xiaozhouX** [comment]
Now i change the action of top job xxx .
```
# arena top job tf-dist-data-auto-scaler-test
INSTANCE NAME                                 STATUS   NODE           GPU(Device Index)  GPU(Duty Cycle)  GPU(Memory MiB)
tf-dist-data-auto-scaler-test-tfjob-ps-0      Running  192.168.0.179  N/A                N/A              N/A
tf-dist-data-auto-scaler-test-tfjob-worker-0  Running  192.168.0.180  0                  0%               1MiB / 12210MiB 
                                                                      1                  0%               1MiB / 12210MiB 
```

Arena top job command remain the action
```
# arena top job
NAME                           STATUS   TRAINER  AGE  NODE           GPU(Requests)  GPU(Allocated)
tf-dist-data-auto-scaler-test  RUNNING  tfjob    20m  192.168.0.180  2              2


Total Allocated GPUs of Training Job:
2   

Total Requested GPUs of Training Job:
2   
```

**Comment by cheyang** [comment]
Thanks! It is awesome!  Will you add some documentation? Thanks.

**Comment by xiaozhouX** [comment]
I add a doc and deploy user guide in this  [commit](https://github.com/kubeflow/arena/pull/74/commits/6bbac5d909e779f0c46edbcd7b5a9af717189acf). 
You can try it. Thanks.

File: kubernetes-artifacts/prometheus/gpu-nodeExpoter.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    metadata:
+      labels:
+        app: node-gpu-exporter
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: aliyun.accelerator/nvidia_count
```
Comment (xieydd) [inline kubernetes-artifacts/prometheus/gpu-nodeExpoter.yml]: Maybe it is not universal.

Comment (xiaozhouX) [reply kubernetes-artifacts/prometheus/gpu-nodeExpoter.yml]: You are right, I updated docs, user need to label their GPU node manually.

File: cmd/arena/commands/gpu_info.go [hunk 2]
```diff
@@ -0,0 +1,212 @@
+package commands
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"encoding/json"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	v12 "k8s.io/api/core/v1"
```
Comment (cheyang) [inline cmd/arena/commands/gpu_info.go]: Please change it to `corev1 "k8s.io/api/core/v1"` , and `metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"`

File: docs/userguide/9-top-job-gpu-metric.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+1\. Deploy a Prometheus
+
+```
+kubectl apply -f kubernetes-artifacts/prometheus/prometheus.yaml
+```
+
+2\. Deploy GPU node exporter
+
+If your cluster is aliyun kubernetes cluster, you can just exec command:
```
Comment (cheyang) [inline docs/userguide/9-top-job-gpu-metric.md]: Please use ACK ( Alibaba Cloud Kubernetes)

**Pull request merged**